sos/cleaner/__init__.py

```diff
@@ -184,6 +184,8 @@ def print_disclaimer(self):
             except KeyboardInterrupt:
                 self.ui_log.info("\nExiting on user cancel")
                 self._exit(130)
+            except Exception as e:
+                self._exit(130, e)
```

Comment:
`130` is specifically for a keyboard interrupt. So I'd say a different exit code should be given, probably `1`?